saleor/graphql/product/mutations/products.py

```diff
@@ -294,16 +294,13 @@ def user_is_allowed(cls, user, input):
         return user.has_perm('product.manage_products')
 
 
-class ProductVariantInput(graphene.InputObjectType):
+class ProductVariantPropertiesInput(graphene.InputObjectType):
```

Comment:
I think the word "properties" may be misinterpreted as relating to product attributes. I would keep the name ProductVariantInput" here. The other name, "ProductVariantCreateInput" is fine.